lib/Borel/Error.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- | Copyright 2013-2015 Anchor Systems, Pty Ltd and Others
```

Comment:
re. the debug flag, I didn't want to pollute the interface unnecessarily. I've removed the flag for the time being though, and just emit debug messages always, perhaps I'll revisit it later.